test/simplegraphs/generators/staticgraphs.jl

```diff
@@ -243,4 +243,36 @@
     Adj = sparse(I, J, V)
     @test Adj == sparse(rg3)
     @test isvalid_simplegraph(g)
+
+    g = @inferred(LadderGraph(5))
+    @test nv(g) == 10 && ne(g) == 13
+    @test isvalid_simplegraph(g)
+    # tests for extreme values
+    g = LadderGraph(0)
+    @test nv(g) == 0 && ne(g) == 0
+    g = LadderGraph(1)
+    @test nv(g) == 2 && ne(g) == 1
+    g = @inferred LadderGraph(-1)
+    @test nv(g) == 0 && ne(g) == 0
+    g = LadderGraph(Int8(63))
+    @test nv(g) == 126 && ne(g) == 187
+    @test eltype(g) == Int8
+    # tests for errors
+    @test_throws InexactError LadderGraph(Int8(64))
```

Comment:
Since isomorphism is still in Experimental, is it a good idea to use here? Testing `LadderGraph(3) == Grid([3,2])` works but only by coincidence on the manner in which the graph is labeled.

Reply:
@mcognetta makes a great point: we should probably not be relying on anything in Experimental for tests. There should be other ways to assure the correctness of the algorithm, even if they're not as efficient / pretty / elegant.